_posts/2022-06-13-connecting-the-pods.md

```diff
@@ -0,0 +1,291 @@
+On the other hand, we also speak of Social BPM when we allude to leveraging the organization’s outward social media resources into internal business processes.
+
+In 2014 Gartner had retitled the first of the mindsets as Design-By-Doing BPM, as placed it thus named on its Hype Cycle, as seen in the illustration below. 
+While supported by advanced BPM technology and social software in order to make process design more visible and holistic (thus guaranteeing the “emergent” design), it also requires a user experience perspective and their willingness to step into such an endeavor so that social networks of employees can be cultivated and mobilized around the idea of bottom-up process improvement efforts.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-06-13-connecting-the-pods/gartnersocialbpm.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+The critical difference between services and products is this more direct co-creation with the customer. 
+They decide where, when, and how the services are delivered. 
+This makes services highly contextual and places the value of services to the customer in the interaction (or customer experience). 
+To draw the similarities with the social web even further: Products can then be considered the end result of a service to the customer and become the physical manifestation or avatar of the service.
+
+It seems like common sense to care about and keep track of customers. 
+But when companies start to grow, a lot of distractions have a way of muddying the waters with other preoccupations, resulting in a decline in customer focus. 
+These distractions can range from a multitude of new opportunities to internal efficiency drives, over-expansion (with product dilution as a result), and so on. 
+This can also lead to an automation of processes to handle the increasing complexity with the caveat of these automations becoming too restrictive, and limiting the creativity and adaptability of your services. 
+Where the author argues this case, I would however stipulate that over-automation is easily mitigated by determining which of your processes are run-of-the-mill and which add value to your unique selling point or differentiator. 
+The former are ideal candidates for such an automation drive (or standardization) whereas the latter are more suited for custom development that allows for easy adaptation. 
+This comes down to finding a balance between efficiency and adaptability/innovation.
+
+Another game changer is the number of companies that go belly up. 
+This is largely due to the complexity of the market increasing, with new technologies popping up all over and disrupting the market in addition to the previously mentioned globalization increases the number of competitors companies have to deal with. 
```

Comment:
This sentence looks a bit odd, does it miss a comma ? I'm not sure what it is but something seems off.

Reply:
fixed